client/src/components/grants/Card.tsx

```diff
@@ -38,7 +41,13 @@ function Card({ projectId }: { projectId: number }) {
 
   return (
     <div className="max-w-sm rounded overflow-hidden shadow-lg my-6">
-      <Link to={grantPath(projectId)}>
+      <Link
+        to={projectPath(
+          projectId.split(":")[0],
+          projectId.split(":")[1],
+          projectId.split(":")[2]
```

Comment:
@DanieleSalatti I would use the data that we get from `getProjectURIComponents` to build the `projectPath`

Reply:
Forgot to refactor these... 🤦🏻‍♂️